modules/web-console/frontend/app/modules/user/User.service.ts

```diff
@@ -80,7 +81,18 @@ export default function UserFactory(
 
             sessionStorage.removeItem('demoMode');
         },
+        normalize(user: Partial<User>): Partial<User> {
```

Comment:
Let's move default values to backend and remove this logic.

Reply:
Done.

Reply:
@akuznetsov-gridgain did you remove the frontend changes?